instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/rpc/MetricsView.java

```diff
@@ -53,6 +53,7 @@ private static Set<AttributeKey> buildServerView() {
     return view;
   }
 
+  @SuppressWarnings("deprecation") // TODO: evaluate strategy for NET_HOST_IP
```

Comment:
Just want to call this out so we don't loose track of it. I punted on this because I was focussed on logs, but hopefully we can address this before 1.19.0.

Reply:
I've fixed this. Changed references from NET_HOST_IP to NET_SOCK_HOST_ADDR, as indicated in the [1.13.0 release notes](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.13.0).